public/locales/sv/signup.json

```diff
@@ -99,6 +99,7 @@
     "labelFirstName": "Förnamn",
     "labelLastName": "Efternamn",
     "labelPhoneNumber": "Telefonnummer",
+    "labelPriceGroup": "Prisgrupp",
```

Comment:
Now that the Finnish term is "Asiakasryhmä", the Swedish and English terms should probably be in line with that ("Kundgrupp" for Swedish and "Customer Group" for English).

I need to do the same change for the back-end as I've used "price group" there too.